src/aiotk/_typing.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+
+
+from types import TracebackType
+from typing import Any, List, Optional, Type
+from typing_extensions import Protocol
+
+
+class AsyncContextManager(Protocol):
+    """User-defined type for `AsyncContextManager`."""
+
+    async def __aenter__(self) -> Any:
+        ...  # pragma: no cover
```

Comment:
**Note to self**: `coverage` needs to be *explicitly* told that our `mypy` protocol definitions are not part of code coverage.